lib/commands/auth/whoami.js

```diff
@@ -0,0 +1,20 @@
+var ui = require('../../ui')
+var TownshipClient = require('../../township')
+
+module.exports = {
+  name: 'whoami',
+  command: whoami,
+  options: []
+}
+
+function whoami (opts) {
+  var client = TownshipClient({server: opts.server})
+
+  client.getConfig(function (err, config) {
```

Comment:
I'm going to need the `currentLogin` but `getLogin` gets just any old login. I can implement this on the `township-client` side